addons/l10n_in/models/company.py

```diff
@@ -37,6 +37,77 @@ class ResCompany(models.Model):
     l10n_in_pan_type = fields.Char(string="PAN Type", compute="_compute_l10n_in_pan_type")
     l10n_in_gst_state_warning = fields.Char(related="partner_id.l10n_in_gst_state_warning")
 
+    # TDS/TCS settings
+    l10n_in_tds_feature = fields.Boolean(string="TDS", inverse="_inverse_l10n_in_tds_feature")
+    l10n_in_tcs_feature = fields.Boolean(string="TCS", inverse="_inverse_l10n_in_tcs_feature")
+    l10n_in_withholding_account_id = fields.Many2one(
+        comodel_name='account.account',
+        string="TDS Account",
+        check_company=True,
+    )
+    l10n_in_withholding_journal_id = fields.Many2one(
+        comodel_name='account.journal',
+        string="TDS Journal",
+        check_company=True,
+    )
+    l10n_in_tan = fields.Char(string="TAN", help="Tax Deduction and Collection Account Number")
+
+    # GST settings
+    l10n_in_is_gst_registered = fields.Boolean(string="Registered Under GST", inverse="_inverse_l10n_in_is_gst_registered")
```

Comment:
Why is this not compute?  Or why do we have this field?  Are not you l10n_in_is_gst_registered if the vat field is filled in?